ProcessesApi/V1/Helpers/ChangeOfNameHelper.cs

```diff
@@ -0,0 +1,16 @@
+using ProcessesApi.V1.Services.Exceptions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ProcessesApi.V1.Helpers
+{
+    public static class ChangeOfNameHelper
+    {
+        public static void ValidateOptionalFormData(Dictionary<string, object> requestFormData, List<string> expectedFormDataKeys)
+        {
+            if (!requestFormData.ContainsKey("firstName") && !requestFormData.ContainsKey("surname") &&
```

Comment:
Sorry, thought I commented this earlier but it didn't go through

You can just do: 
```csharp
expectedFormDataKeys.Any(x => requestFormData.ContainsKey(x))
```
Then it'll be generic and you won't have to put it in a dedicated helper class, can just go into processhelpers